src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -1110,6 +1096,7 @@ public void run() {
 
 		if (MovementHandler.isStopped(player) || Bloodbending.isBloodbent(player) || Suffocate.isBreathbent(player)) {
 			event.setCancelled(true);
+			Bukkit.broadcastMessage("the 3 combo thingy set the event as cancelled"+MovementHandler.isStopped(player)+Bloodbending.isBloodbent(player)+Suffocate.isBreathbent(player));

```

Comment:
This is an extremely unprofessional debug message. not to mention it doesn't tell us anything useful and had I not seen this in your pull request I as a developer would not be able to identify what this message means.

Reply:
I don't think he meant to include his haha

Reply:
Yes I didn't mean to